src/metabase/pulse/render.clj

```diff
@@ -100,6 +100,7 @@
            :area
            :bar
            :combo
+           :pie2
```

Comment:
This name will also change later in the stack when we remove the dynamic chart used for testing